README.md

```diff
@@ -0,0 +1,33 @@
+## Screens
+
+### Development
+
+#### Prerequisites
+- [Docker](https://www.docker.com/docker-toolbox)
+- [Heroku Toolbelt](https://toolbelt.heroku.com/)
+- Heroku Docker plugin -- `heroku plugins:install heroku-docker`
+
+#### Setting up development environment
+- Clone the repository -- `git clone git@github.com:ftlabs/screens.git`
+- Change in repository directory -- `cd screens`
+- If running OS X or Windows, [follow these steps for creating a virtual machine for Docker](#creating-a-virtual-machine-for-docker) -- This is a section in this file.
```

Comment:
would prefer this was listed in-place, indented, rather than as a separate section in da doc. Is not totally clear reading back and forth between the section it is first mentioned in, and then this extra section.